docs/cse/integrations/automation-service.md

```diff
@@ -0,0 +1,209 @@
+:::
+
+## Permissions for the CSE Automation Service
+
+Access to the CSE Automation Service is controlled by permissions (capabilities) in the Sumo Platform:
+* Cloud SIEM Enterprise > Configuration > View Automations
+* Cloud SIEM Enterprise > Configuration > Manage Automations 
+* Cloud SIEM Enterprise > Configuration > Execute Automations
+* Cloud SOAR > View Cloud SOAR
+* Cloud SOAR > Automation Playbooks > Access
+* Cloud SOAR > Automation Playbooks > Configure
+
+Note that in order to interact with most of the features described on this page, you must have at least View Automations, View Cloud SOAR, and Access Playbooks permissions.
+
+## Accessing the CSE Automation Service
+
+The Automation page can be accessed in the **Configuration** menu under **Integrations**:
+
+<img src={useBaseUrl('img/cse/automations-config-menu.png')} alt="Automation menu option" width="170"/>
+
+From here, you can manage Automations and access the CSE Automation Service UI to manage Playbooks, Actions, and Integrations. To access the CSE Automation Service UI, click **Manage Playbooks**:
+
+<img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="450"/>
+
+You will see the Playbooks list in the CSE Automation Service UI:
+
+<img src={useBaseUrl('img/cse/automations-playbook-list.png')} alt="Automation Playbook list" width="600"/>
+
+To learn more about managing Playbooks, Actions, and Integrations, read [Cloud SOAR Automation](/docs/cloud-soar/automation/). In addition, some documentation (including the documentation for building custom Integrations and using the Bridge) is included in the UI. It can be accessed by clicking the Help icon in the upper-right corner of the screen.
+
+:::note
+Beta: Some notes about Playbooks, Integrations and Actions in the Beta that may differ from the current Cloud SOAR documentation:
+* Playbooks type must be **CSE**.
+* The CSE Automation Service only supports automated Enrichment, Notification, and Custom Action types at this time. 
+* Actions can run "on premise" via a Bridge or can run directly through the Cloud. For security and performance reasons, only Certified Integrations/Actions can run directly through the Cloud; custom Actions must run "on premise".
+* Cloud SOAR App Central, where you can browse the full Integration and Playbook catalog, is not yet connected to the CSE Automation Service. Contact Sumo Logic if you are interested in a specific Integration that is not yet included with the CSE Automation Service.
+:::
+
+## Configuring an Automation
+
+To create a new Automation:
+1. Click the gear icon at the top of the CSE UI and choose **Automation** under **Integrations**. 
+1. Click **New Automation**. 
+   (To modify an existing Automation, click on the edit icon for the corresponding Automation.)
+
+   <img src={useBaseUrl('img/cse/automations-new.png')} alt="New Automation" width="560"/>
+
```

Comment:
Our style is to close gaps in sequential steps. Otherwise you'll get giant spaces between each step like this
<img width="881" alt="Screenshot 2023-03-31 at 3 43 07 PM" src="https://user-images.githubusercontent.com/56411016/229245491-75f3caf3-fc03-4e36-9ebb-f753f6a00b09.png">


```suggestion
1. Click **New Automation**.  (To modify an existing Automation, click on the edit icon for the corresponding Automation.)</br><img src={useBaseUrl('img/cse/automations-new.png')} alt="New Automation" width="560"/>
```